### DISCUSSION TITLE
Can I run PlexCleaner from multiple machines, all pointing to the same samba share to process faster?

### DISCUSSION THREAD

**Discussion by asophila** [ptr727/PlexCleaner#155] [Q&A]
I have a lot of media on a machine that's kind of old. So, running the process there makes the plex reproduction very laggy.
I have another 3 computers on my home network, and i've managed to share mi media through smb, so I can mount that on my other machines.
I can run the process in any of them, but... is it possible to run them in parallel? Sidecar files would allow that? It will be assigned a different file to each machine?

**Comment by ptr727** [comment]
Distributed processing is not supported, i.e. there is no filesystem / quorum locking while processing a file from multiple systems.  
You will need to select different files for processing on different machines, e.g. move your unprocessed files into a directory per machine, once finished processing move the files back.
